x-pack/platform/plugins/shared/alerting/server/routes/gaps/apis/gap_auto_fill_schedule/create/create_auto_fill_scheduler_route.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { IRouter } from '@kbn/core/server';
+import type { GapAutoFillSchedulerResponseV1 } from '../../../../../../common/routes/gaps/apis/gap_auto_fill_scheduler';
+import { gapAutoFillSchedulerBodySchemaV1 } from '../../../../../../common/routes/gaps/apis/gap_auto_fill_scheduler';
+import type { ILicenseState } from '../../../../../lib';
+import { verifyAccessAndContext } from '../../../../lib';
+import type { AlertingRequestHandlerContext } from '../../../../../types';
+import { transformRequestV1, transformResponseV1 } from './transforms';
+import { INTERNAL_ALERTING_GAPS_AUTO_FILL_SCHEDULER_API_PATH } from '../../../../../types';
+import { DEFAULT_ALERTING_ROUTE_SECURITY } from '../../../../constants';
+
+export const createAutoFillSchedulerRoute = (
+  router: IRouter<AlertingRequestHandlerContext>,
+  licenseState: ILicenseState
+) => {
+  router.post(
+    {
+      path: INTERNAL_ALERTING_GAPS_AUTO_FILL_SCHEDULER_API_PATH,
+      security: DEFAULT_ALERTING_ROUTE_SECURITY,
+      options: { access: 'internal' },
+      validate: {
+        body: gapAutoFillSchedulerBodySchemaV1,
+      },
+    },
+    router.handleLegacyErrors(
+      verifyAccessAndContext(licenseState, async function (context, req, res) {
+        const alertingContext = await context.alerting;
+        const rulesClient = await alertingContext.getRulesClient();
```

Comment:
I've got a few more questions on validation:

1. Don't we have max gap fill as 90d? API allows to pass  range as ` "gap_fill_range": "now-100d". Should it be also limited?

2. what field scope for? 
I am able to pass any value there and API creates scheduler task